addons/sale_management/models/sale_order_template.py

```diff
@@ -16,6 +16,7 @@ class SaleOrderTemplate(models.Model):
 
     name = fields.Char(string="Quotation Template", required=True)
     note = fields.Html(string="Terms and conditions", translate=True)
+    so_prefix = fields.Char('Reference Code', help="If defined, all the documents using this template will start with this prefix. Not mandatory")
```

Comment:
```suggestion
    so_prefix = fields.Char('Reference Code', help="If defined, all the documents using this template will start with this prefix.")
```

No need to specify it, if it was mandatory, they would see it in the view ;)